twilight-validate/src/request.rs

```diff
@@ -920,7 +920,7 @@ pub fn scheduled_event_description(description: impl AsRef<str>) -> Result<(), V
 /// [`ScheduledEventGetUsers`]: ValidationErrorType::ScheduledEventGetUsers
 /// [this documentation entry]: https://discord.com/developers/docs/resources/guild-scheduled-event#get-guild-scheduled-event-users-query-string-params
 pub const fn scheduled_event_get_users(limit: u16) -> Result<(), ValidationError> {
-    if limit <= SCHEDULED_EVENT_GET_USERS_MIN && limit >= SCHEDULED_EVENT_GET_USERS_MAX {
+    if limit < SCHEDULED_EVENT_GET_USERS_MIN || limit > SCHEDULED_EVENT_GET_USERS_MAX {
```

Comment:
```suggestion
    if limit >= SCHEDULED_EVENT_GET_USERS_MIN && limit <= SCHEDULED_EVENT_GET_USERS_MAX {
```
Please also add a test case for this function.